tests/avocado/test_numa_hotplug.py

```diff
@@ -20,7 +21,12 @@ class TestNumaHotplug(test.QemuTest):
     """
 
     def setUp(self):
-        self.request_image()
+        # Getting latest Fedora Cloud Image
+        self.image = vmimage.get('Fedora')
+
+        # Fedora Cloud Image has default username 'fedora'
+        self.vm.add_image(self.image.path, 'fedora', 'fedora', snapshot=False)
```

Comment:
Same comment here WRT the boilerplate code being necessary (shouldn't be unless the test writer wants to deviate from the norm).